README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Current Released  Version
 
-Version `1.0.0`
+Version `1.0.1`
```

Comment:
We need to create / document change log regarding what changed since last release. This is useful for users to decide if they should upgrade to new version

Reply:
Created https://github.com/cloudspannerecosystem/cassandra-to-spanner-proxy/pull/16, PTAL